### ISSUE TITLE
scalastyle section of nightly.sh doesn't work

### ISSUE THREAD

**Issue by SethTisue** [NetLogo/NetLogo-Headless#51]
at the end of each nightly run we see:

```
[error] Expected ID character
[error] Not a valid command: parserJvm
[error] Expected project ID
[error] Expected configuration
[error] Expected ':' (if selecting a configuration)
[error] Expected key
[error] Not a valid key: parserJvm
[error] parserJvm/scalastyle
[error]          ^
```

but then the error doesn't propagate out and fail the build.

(@mrerrormessage?)

**Comment by mrerrormessage** [comment]
I'll look into it. I believe this broke in the upgrade from scala-js 0.5 to 0.6. Due to the way this step is done in `nightly.sh`, the command can return a non-zero exit code and the build can go on.

**Comment by mrerrormessage** [comment]
Fixed in #53 .
